packages/premint-sdk/src/generated/premint-api-types.ts

```diff
@@ -0,0 +1,363 @@
+/**
+ * This file was auto-generated by openapi-typescript.
```

Comment:
super cool!
what if this was gitignored?

Reply:
This is based off an api response and a tool that may change its output format. It makes sense then to include in the repo since this file is pretty brittle and if it changes we can manually update.